content/powershell/concepts/variables/variables.md

```diff
@@ -1,12 +1,68 @@
 ---
 Title: 'Variables'
-Description: ''
+Description: 'Variables are a way to store and manipulate data. There are multiple types including environment variables which can be created in variety of ways.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
+  - 'PowerShell'
```

Comment:
```suggestion
  - 'Bash/Shell'
```
This is the subject as listed in documentation/subjects.md. You could add "PowerShell" to the subjects.md file, but it might appear redundant.